### PR TITLE
Ed: integration tests

### PR DESCRIPTION
**Pull request by antoine-de**
use the new Eitri module (based on docker for the DB) to handle ed integration tests

`make` and `make test` do not build/run the ed integration tests

to run them call `make docker_test`

Note: you need of course to have docker installed to run them.
if you haven't added your user to the docker group you need the root privilege to run docker

### CODE REVIEW

File: source/ed/CMakeLists.txt [hunk 1]
```diff
@@ -49,3 +49,6 @@ add_executable(synonym2ed synonym2ed.cpp)
 target_link_libraries(synonym2ed transportation_data_import connectors)
 
 INSTALL_TARGETS(/usr/bin/ gtfs2ed osm2ed ed2nav fusio2ed fare2ed geopal2ed poi2ed synonym2ed)
+
+# Ed integration tests with docker, build with the docker_test target and not with the defaut target not to have a strong docker dependency
```
Comment (pbougue) [inline source/ed/CMakeLists.txt]: Line length :cry:

Comment (pbougue) [inline source/ed/CMakeLists.txt]: defauLt

**Comment by pbougue** [comment]
This "WIP" commit deserves squash or... whip.

**Comment by antoine-de** [comment]
:fu:

**Comment by pbougue** [comment]
:

File: source/ed/CMakeLists.txt (hunk 1 above)
Comment (antoine-de) [inline source/ed/CMakeLists.txt]: done but you fail on the close/reopen keeps the comment visible :wink:

**Pull request merged**